requirements.txt

```diff
@@ -45,6 +45,9 @@ git+https://github.com/tsauerwein/ColanderAlchemy.git@c2corg
 # needs: https://github.com/mozilla-services/cornice/pull/359
 git+https://github.com/tsauerwein/cornice.git@nested-none-2.1.0-c2corg
 
+# c2c_markdown repo
+git+https://github.com/c2corg/c2c_markdown.git@master
```

Comment:
Do we really need it from master? I'd rather reference a tag here.

Reply:
A commit is not possible, because for a reason I ignore, pip won't install dependancies.

It's a commit for common, but it does no fail only because v6_common has not dependancies...

I propose to let it as is for a couple of month, because I will probably modify this dep.

As we will be in a more stable situation, we'll use a tag (if it works) or a dedicated branch.

Reply:
I would prefer to create a version in c2c_markdown. And create as many as needed later on. You can't rely on a moving target, so a tag is appropriate.

Reply:
See the key point in my previous response : 

> It's a commit for common, but it does no fail only because v6_common has not dependancies...

Reply:
I am not sure why it didn't work for you. But for pip it makes no difference if you use a branch name, commit or tag. For testing, I set your latest commit on master in this branch:
https://github.com/c2corg/v6_api/commit/f042a8314932f9e6addc59ebfe59b231c5fbd259

And the Travis build passed:
https://travis-ci.org/c2corg/v6_api/jobs/470141545

Reply:
Well, to be frank, I can't explain why I came to this conclusion. Your test is crystal clear...

For the next few week, as I'll probably iterate a lot to fix small bugs, would you mind if we  keep it on master branch?

Then, when we'll be in production, we'll set it to a commit, or a branch (I prefer this last option, the branch name is more explicit).

Reply:
I insist, rather than a branch, the tag is the way to go. A branch is still a moving target.
I you know I don't understand why you don't want to create tags :) but anyway, as long as there is a tag when merged into master, I'm happy.

Reply:
I wanted to use master to keep a moving target during these week. But in fact, it seems taht docker image doesn't update itself, as it consider it still contains the dependancy.

So I just commited this line with an explicit commit : 7012ce6

This said, I also agree that a tag is better than a commit : more explicit. I just saw your message after my last commit 5 minutes ago, sorry.